src/App.tsx

```diff
@@ -20,17 +24,83 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: 2,
+    frameSize: 3,
+    itemWidth: 130,
+    animationDuration: 1000,
+    // infinite: false,
+  };
+
+  handleChange = (event: ChangeEvent<HTMLInputElement>, key: string) => {
+    const input = event.target.value;
+
+    this.setState(state => (
+      {
+        ...state,
+        [key]: +input,
+      }
+    ));
   };
 
   render() {
-    const { images } = this.state;
+    const {
+      images,
+      step,
+      frameSize,
+      itemWidth,
+      animationDuration,
+      // infinite,
+    } = this.state;
 
     return (
       <div className="App">
         {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
+        <h1 data-cy="title">Carousel with {images.length} images</h1>
 
-        <Carousel />
+        <Carousel
+          images={images}
+          step={step}
+          frameSize={frameSize}
+          itemWidth={itemWidth}
+          // animationDuration={1000}
+          // infinite={infinite}
+        />
+        <div>
+          <label htmlFor="itemWidth">Item Width:</label>
+          <input
+            type="number"
+            value={itemWidth}
+            onChange={(event) => this.handleChange(event, 'itemWidth')}
+            id="itemWidth"
```

Comment:
```suggestion
            onChange={this.handleChange}
            id="itemWidth"
```
You can rewrite your handleChange function so it takes key (2nd argument) from event. You could access it with event.target.id.